portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,6 +15,9 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import com.google.gson.Gson;
```

Comment:
My recollection of the [Google rules](https://google.github.io/styleguide/javaguide.html#s3.3-import-statements) is that these should be sorted alphabetically.

Reply:
I didn't know that, I will go ahead and fix it!